cmd/ocm-backplane/logout/logout.go

```diff
@@ -26,65 +27,74 @@ var LogoutCmd = &cobra.Command{
 
 func runLogout(cmd *cobra.Command, argv []string) error {
 
-	rc, err := utils.ReadKubeconfigRaw()
-	if err != nil {
-		return err
+	// Logout specific cluster
+	if len(argv) == 1 {
+		err := login.RemoveClusterKubeConfig(argv[0])
+		if err != nil {
+			return err
+		}
+		fmt.Printf("Logged out from backplane: %s\n", argv[0])
+	} else {
+		rc, err := utils.ReadKubeconfigRaw()
```

Comment:
Can you please elaborate what `rc` means here?

Reply:
`rc` mean ReadConfig from the default config location( ~/.kube/config).
I added a comment to the code.